plugins/pulp_docker/plugins/importers/sync.py

```diff
@@ -89,6 +141,34 @@ def generate_download_requests(self):
         for unit in self.step_get_local_blobs.units_to_download:
             yield self.index_repository.create_blob_download_request(unit.digest)
 
+    def v1_generate_download_requests(self):
```

Comment:
Similarly, I optimized here for minimal code change. I think that's a good idea, but will leave it as-is for now. And I do think there's value in keeping it the way it is, as a method on a step, to follow the pattern of other similar methods.